### PR TITLE
Add OptaWebs tests

### PR DESCRIPTION
**Pull request by dupliaka**
To check the product of employee rostering and the vehicle routing of the products version.

The same functionality is tested in kogito community pipeline.
However we need to check integration of product version of RedHat distribution.

### CODE REVIEW

File: run-tests-with-build-optaplanner/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-run-tests-with-build-optaplanner
+    run-tests-with-build-optaplanner
```
Comment (jstastny-cz) [inline run-tests-with-build-optaplanner/README.md:1]: is this intentional?

File: pom.xml [hunk 2]
```diff
@@ -34,7 +35,7 @@
 
     <kie.download.build.version>7.48.0.Final</kie.download.build.version>
     <kogito.download.build.version>2.0.0-SNAPSHOT</kogito.download.build.version>
-    <optaplanner.download.build.version>8.4.1.Final</optaplanner.download.build.version>
+    <optaplanner.download.build.version>8.3.0.redhat-2021041404</optaplanner.download.build.version>
```
Comment (jstastny-cz) [inline pom.xml:38]: Wouldn't it make sense here to default to an already released version? (if this one is not of course)

**Review by jstastny-cz — APPROVED** [review]
LGTM, just minor comments.

File: pom.xml (hunk 2 above)
Comment (dupliaka) [reply pom.xml:38]: That is not clear for me, could you point me out which group do you mean? 
http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/groups/

File: run-tests-with-build-optaplanner/README.md (hunk 1 above)
Comment (dupliaka) [reply run-tests-with-build-optaplanner/README.md]: Fixed 76cb1f6c

File: pom.xml (hunk 2 above)
Comment (jstastny-cz) [reply pom.xml:38]: I meant if there's an externally accessible version, but there's probably not. Disregard.

**Review by jstastny-cz — APPROVED** [review]

**Comment by dupliaka** [comment]
@jstastny-cz could you please merge it then?

File: pom.xml (hunk 2 above)
Comment (dupliaka) [reply pom.xml:38]: @jstastny-cz please clarify what is externally accessible means in the context of testing product versions

**Pull request merged**